gtsfm/bundle/bundle_adjustment.py

```diff
@@ -275,6 +275,17 @@ def __cameras_to_model(
         proxy for this function."""
         cameras: Set[int] = set(initial_data.get_valid_camera_indices())
 
+        # Add all relative pose priors have atleast one node in the GTSFM's data valid cameras.
+        for i1, i2 in relative_pose_priors.keys():
+            if i1 in cameras or i2 in cameras:
+                cameras.add(i1)
+                cameras.add(i2)
+
+        # Hilti specific hack to add the backbone of CAM2-CAM2
+        # TODO(Ayush): move this to a hilti specific branch
+        for i in range(2, initial_data.number_images(), 5):
+            cameras.add(i)
+
```

Comment:
@dellaert does this look good? These two code blocks are the ones which have unitialized logic.

Reply:
We should not *have* hilti-specific branches, only hilti-specific classes and flags. Can we not find out which ones are missing here in general?